addons/dynamic-mount/ossfs/docker/umount.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xe
+
+echo "umount.sh: MOUNT_POINT ${MOUNT_POINT}"
+for folder in ${MOUNT_POINT}/*; do
+    if [ -d "$folder" ]; then
+        echo "umount_all: umount ${folder}"
+        umount ${folder}
+        rm -rf ${folder}
```

Comment:
Better use `rmdir ${folder}`. `rm -rf` may mistakenly remove all the data in the underlying storage. It's very dangerous.

Reply:
Ok.